deployment/single-server/app.yml

```diff
@@ -0,0 +1,122 @@
+---
+
+# Invoke using:
+#
+#     ansible-playbook app.yml --become --ask-become-pass
+#
+
+- name: Objects API
+  hosts: all
+
+  collections:
+    - maykinmedia.commonground
+
+  vars:
+    # Application
+    django_app_docker_name_prefix: objects
+    django_app_docker_domain: TODO
+    django_app_docker_secret_key: 'TODO'
+    django_app_docker_package_name: objects
+    django_app_docker_https: false
+
+    ## Application database settings
+    django_app_docker_db_host: TODO
+    django_app_docker_db_port: 5432
+    django_app_docker_db_name: objects
+    django_app_docker_db_username: objects
+    django_app_docker_db_password: TODO
+
+    ## Application docker container
+    django_app_docker_version: latest
+    django_app_docker_image_name: maykinmedia/objects-api
+    django_app_docker_replicas: 1
+    django_app_docker_port_start: 12000
+
+    django_app_docker_volumes:
+      - name: "{{ django_app_docker_name_prefix }}-media"
+        mount: "/app/media"
+        mode: "rw"
+        owner: 1000
+      - name: "{{ django_app_docker_name_prefix }}-private-media"
```

Comment:
I don't think we use private media. If not, this whole block can be removed since the role has the media volume as default.